app/templates/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"/>
+    <link rel="stylesheet" href="/app/static/index.css">
+    <title>Document</title>
+</head>
+<body>
+    <main>
+        <header>
+            <div class="menu">
+                <ul class="logo">
+                    <li id="left"><a href="#">My Logo</a></li>
+                </ul>
+                <ul class="hamburger-menu">
+                    <li><a href="#"><div class="menu-icon"></div></a></li>
+                    <li><a href="#"><div class="menu-icon"></div></a></li>
+                    <li><a href="#"><div class="menu-icon"></div></a></li>
+                </ul>
+            </div>
+        </header>
+        <section class="headline">
+            <div class="background-image">
+                <div class="header-text">
+                    <h1>I’m Mariane <br> Glad to see you!</h1>
+                </div>
+                <div class="about-text">
+                    <p>I’m a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.
+                    </p>
+                </div>
+                <div class="social-media">
+                    <h4>LET’S CONNECT</h4>
+                    <ul class="icons">
+                        <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                        <li><a href="#"></a><i class="fa-brands fa-linkedin-in"></i></li>
+                        <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
+                        <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+                        <li><a href="#"><img src="/app/static/images/Vector.svg" alt=""></a></li>
+                    </ul>
```

Comment:
- [OPTIONAL] Great job on making the required changes. You are the greatest. It is highly recommended to align your social icons to the left. So that it would perfectly resemble the provided Figma design.

Reply:
Thanks @Qoosim